extensions/parquet-extensions/example/wikipedia_hadoop_parquet.json

```diff
@@ -0,0 +1,75 @@
+{
+  "dataSchema": {
+    "dataSource": "wikipedia",
+    "parser": {
+      "type": "parquet",
+      "parseSpec": {
+        "format": "json",
+        "timestampSpec": {
+          "column": "timestamp",
+          "format": "auto"
+        },
+        "dimensionsSpec": {
+          "dimensions": [
+            "page",
+            "language",
+            "user",
+            "unpatrolled",
+            "newPage",
+            "robot",
+            "anonymous",
+            "namespace",
+            "continent",
+            "country",
+            "region",
+            "city"
+          ],
+          "dimensionExclusions": [],
+          "spatialDimensions": []
+        }
+      }
+    },
+    "metricsSpec": [
+      {
+        "type": "count",
+        "name": "count"
+      },
+      {
+        "type": "doubleSum",
+        "name": "added",
+        "fieldName": "added"
+      },
+      {
+        "type": "doubleSum",
+        "name": "deleted",
+        "fieldName": "deleted"
+      },
+      {
+        "type": "doubleSum",
+        "name": "delta",
+        "fieldName": "delta"
+      }
+    ],
+    "granularitySpec": {
+      "type": "uniform",
+      "segmentGranularity": "DAY",
+      "queryGranularity": "NONE",
+      "intervals": ["2013-08-31/2013-09-01"]
+    }
+  },
+  "ioConfig": {
+    "type": "hadoop",
+    "inputSpec": {
+      "type": "static",
+      "paths": "examples/indexing/wikipedia_data.json"
+    },
+    "segmentOutputPath": "\/tmp\/segments"
+  },
```

Comment:
how did you test this example? don't you have to specify your input format here ? also "paths" refer to a json file.